doc/lua/methods.md

```diff
@@ -122,7 +122,7 @@ Set the settings with a table of keys:
 - speed: GS_VERYSLOW, GS_SLOW, GS_NORMAL, GS_FAST, GS_VERYFAST
 - objective: GO_NONE, GO_CONQUER3_4, GO_TOTALDOMINATION
 - startWares: SWR_VLOW, SWR_LOW, SWR_NORMAL, SWR_ALOT
-- fow: EXP_DISABLED, EXP_CLASSIC, EXP_FOGOFWAR, EXP_FOGOFWARE_EXPLORED
+- fow: EXP_DISABLED, EXP_CLASSIC, EXP_FOGOFWAR, EXP_FOGOFWAREXPLORED
```

Comment:
shouldnt this not be called `EXP_FOGOFWAR_EXPLORED` to make it readable? (so "only" fix the typo)

Reply:
I wanted to a) keep it consistent with everything else (`VERYFAST` not `VERY_FAST`, ...) and b) allow auto-generation of the names by simply upper-casing the names which saves quite some work (and can even be more automated than it is now by using the output operators)